nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -158,6 +158,16 @@ The module update takes care of the new config syntax and the data itself (user
 
 ## Nixpkgs internals {#sec-release-23.11-nixpkgs-internals}
 
+- The `django` alias in the python package set was upgraded to Django 4.x. Applications that consume Django should always pin their python
+  environment to a compatible major version, so they can move at their own pace.
+  ```nix
+  python = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_3;
+    };
+  };
+  ```
+
```

Comment:
PTAL @Shawn8901

Reply:
thank you very much! :) that also double adds directly the best practice to not get broken, so lgtm.